### PR TITLE
Project description

### PR DESCRIPTION
**Pull request by usoltsev37**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+* The settings of a musical instrument (for example, a synthesizer) for a specific sound 
+
+Project presentation is available [here](https://docs.google.com/presentation/d/1CMpOUXObjyO55Z2z2sKBcBahaKF9QfR-16G77WDwUFA/edit?usp=sharing)
+
+## Installation
+
+To run the server locally, please, clone our repository and in the root directory of the project run the following commands
+```bash
+$ sudo docker compose build
+$ sudo docer compose up -d
```
Comment (TanVD) [inline README.md:17]: Probably, `docker`?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+```
+Download the ```app-debug.apk``` file and open it on your Android device
+
+## Usage
+Users of the application can perform the following actions:
+* Login user by google mail
```
Comment (TanVD) [inline README.md:23]: User can login user?

File: README.md (hunk 1 above)
Comment (Readrid) [reply README.md:17]: Fixed in #6 PR

File: README.md (hunk 2 above)
Comment (Readrid) [reply README.md:23]: Fixed in #6 PR

**Pull request merged**